test/controllers/confirmation.controller.spec.unit.ts

```diff
@@ -128,6 +200,14 @@ describe("Company no longer required confirmation screen tests", () => {
     expect(resp.text).toContain("BT2 8BG");
     expect(resp.text).toContain(PAGE_TITLE);
   });
+});
+
+describe("Company still required confirmation screen tests", () => {
+
+  const mockCacheService = loadSession as jest.Mock;
```

Comment:
you can move these 4 mocks so they are declared before the first describe(), you won't have to repeat them then

Reply:
and delete the others from the other describe()

Reply:
Okay, will do.